mods/ra/maps/allies-06b/map.yaml

```diff
@@ -0,0 +1,851 @@
+		Owner: USSR
+		Facing: 191
+	SubPatrol4_1: waypoint
+		Location: 79,51
+		Owner: Neutral
+	SubPatrol4_2: waypoint
+		Location: 102,51
+		Owner: Neutral
+	Patrol5Sub: ss
+		Owner: USSR
+		Location: 94,75
+		Facing: 125
+	SubPatrol5_1: waypoint
+		Location: 94,94
+		Owner: Neutral
+	SubPatrol5_2: waypoint
+		Location: 94,75
+		Owner: Neutral
+	Mammoth1: 4tnk
+		Location: 69,43
+		Owner: USSR
+		Facing: 63
+	Mammoth2: 4tnk
+		Location: 43,40
+		Owner: USSR
+		Facing: 63
+	Mammoth3: 4tnk
+		Location: 54,37
+		Owner: USSR
+		Facing: 63
+	TnkPatrol1: waypoint
+		Location: 69,43
+		Owner: Neutral
+	TnkPatrol2: waypoint
+		Location: 69,55
+		Owner: Neutral
+	TnkPatrol3: waypoint
+		Location: 56,55
+		Owner: Neutral
+	TnkPatrol4: waypoint
+		Location: 41,44
+		Owner: Neutral
+	TnkPatrol5: waypoint
+		Location: 42,37
+		Owner: Neutral
+	TnkPatrol6: waypoint
+		Location: 62,37
+		Owner: Neutral
+	DefenseFlame1: ftur
+		Location: 51,91
+		Owner: USSR
+	DefenseFlame2: ftur
+		Location: 47,91
+		Owner: USSR
+	DefBrl1: brl3
+		Location: 51,90
+		Owner: USSR
+	DefBrl2: brl3
+		Location: 46,89
+		Owner: USSR
+	Patrol_1_e1: e1
+		Location: 40,44
+		Owner: USSR
+		Facing: 63
+		SubCell: 1
+	Patrol_1_dog: dog
+		Location: 40,43
+		Owner: USSR
+		Facing: 63
+		SubCell: 1
+	Patrol_2_e1: e1
+		Location: 41,37
+		Owner: USSR
+		Facing: 63
+		SubCell: 1
+	Patrol_2_dog: dog
+		Location: 41,37
+		Owner: USSR
+		Facing: 63
+		SubCell: 3
+	Patrol_3_e1: e1
+		Location: 84,43
+		Owner: USSR
+		Facing: 63
+		SubCell: 2
+	Patrol_3_dog: dog
+		Location: 86,43
+		Owner: USSR
+		Facing: 63
+		SubCell: 0
+	Patrol_4_e1: e1
+		Location: 67,51
+		Owner: USSR
+		Facing: 95
+		SubCell: 1
+	Patrol_4_dog: dog
+		Location: 66,50
+		Owner: USSR
+		Facing: 95
+		SubCell: 3
+	Patrol1: waypoint
+		Location: 85,43
+		Owner: Neutral
+	Patrol2: waypoint
+		Location: 66,50
+		Owner: Neutral
+	Patrol3: waypoint
+		Location: 53,55
+		Owner: Neutral
+	Patrol4: waypoint
+		Location: 46,49
+		Owner: Neutral
+	Patrol5: waypoint
+		Location: 40,44
+		Owner: Neutral
+	Patrol6: waypoint
+		Location: 41,37
+		Owner: Neutral
+	Patrol7: waypoint
+		Location: 54,37
+		Owner: Neutral
+	Patrol8: waypoint
+		Location: 71,43
+		Owner: Neutral
+	tank1: 3tnk
+		Location: 81,56
+		Owner: USSR
+		Facing: 95
+	tank2: 3tnk
+		Location: 77,54
+		Owner: USSR
+		Facing: 95
+	tank3: 3tnk
+		Location: 64,43
+		Owner: USSR
+		Facing: 159
+	tank4: 3tnk
+		Location: 45,43
+		Owner: USSR
+		Facing: 127
+	tank5: 3tnk
+		Location: 51,56
+		Owner: USSR
+	tank6: v2rl
+		Location: 46,41
+		Owner: USSR
+		Facing: 95
+	tank7: v2rl
+		Location: 62,42
+		Owner: USSR
+		Facing: 159
+	BeachDog: dog
+		Location: 75,54
+		Owner: USSR
+		Facing: 95
+		SubCell: 1
+	BeachPatrol1: waypoint
+		Owner: Neutral
+		Location: 68,59
+	BeachPatrol2: waypoint
+		Owner: Neutral
+		Location: 75,54
+	BeachPatrol3: waypoint
+		Location: 77,58
+		Owner: Neutral
+
+Rules: ra|rules/campaign-rules.yaml, ra|rules/campaign-tooltips.yaml, rules.yaml
```

Comment:
Could you add `ra|rules/campaign-palettes.yaml`, this was recently added and changes the colors to how it was in the original game. See [here](https://github.com/OpenRA/OpenRA/pull/14277/files#diff-43c357dd05c0b8b50f3a9683cb95898bR443).